mobify/progressive-web-scaffold
# Shipping email section

This PR adds some functionality to the email section on the checkout shipping page. Specifically on blur it will check if the email address is known, and show the sign in form if it is. It also adds submission handling for the sign in form.

 **JIRA**: https://mobify.atlassian.net/browse/WEB-1111
 **Linked PRs**: #292 

## Changes
- Added onBlur handler to email address field
- Added call to API to check if email address is known
- Added submit handling for the sign in form

## How to test-drive this PR
- Run `npm run dev`
- Preview [Merlin's Potions](https://preview.mobify.com/?url=https%3A%2F%2Fwww.merlinspotions.com%2F&site_folder=https%3A%2F%2Flocalhost%3A8443%2Floader.js&disabled=0&domain=&scope=1)
- add an item to the cart (as a logged out user)
- navigate to: https://www.merlinspotions.com/checkout/
- enter an email address that has an account associated with it 
- sign in on the form that appears
